researchapp/services/logging.py

```diff
@@ -50,9 +50,13 @@ def _clean(data):
         - method
         - url
     """
-    valid = ['body', 'headers', 'method', 'url']
+    valid = ['body', '_content', 'headers', 'method', 'url']
     data = {k: v for k, v in vars(data).items() if k in valid}
 
     data['headers'] = dict(data['headers'])
 
+    if '_content' in data:
+        data['body'] = data['_content'].decode('utf-8')
```

Comment:
Will track in https://github.com/sync-for-science/tracking/issues/45 and merge this PR now.